crates/starknet/src/transactions.rs

```diff
@@ -159,8 +160,14 @@ impl StarknetTransaction {
     pub fn get_receipt(&self) -> DevnetResult<TransactionReceipt> {
         let transaction_events = self.get_events();
 
+        let mut transaction_messages: Vec<MessageToL1> = vec![];
+        for message in self.get_l2_to_l1_messages().into_iter() {
+            transaction_messages.push(message.try_into()?);
+        }
+
         let mut common_receipt = self.inner.create_common_receipt(
             &transaction_events,
+            &transaction_messages,
```

Comment:
Due to the argument added by #233, I've now clippy complaining about too many arguments on this function.
What is the strategy you want to adopt? We silent the warning for now? Or do you prefer a rework of the arguments?